prow/cmd/deck/static/spyglass/spyglass.ts

```diff
@@ -99,6 +113,39 @@ function toggleExpansion(bodyId: string, expanderId: string): void {
   }
 }
 
+// given a string containing ansi formatting directives, return a new one
+// with designated regions of text marked with the appropriate color directives,
+// and with all unknown directives stripped
+function ansiToHTML(orig: string) {
```

Comment:
:robot: `Avoid implicit any by always specifying return types` 

```suggestion
function ansiToHTML(orig: string): string {
```